app/helpers/projects_helper.rb

```diff
@@ -1,2 +1,14 @@
+# frozen_string_literal: true
+
 module ProjectsHelper
+  def total_project_earning(project)
+    project.payments.sum(:amount)
+  end
+
+  def control_links(resource)
+    return unless current_user&.manager?
+
+    link_to('Edit', edit_project_path(resource), class: 'btn btn-primary btn-round') +
```

Comment:
go for something like

capture do
      concat link_to("Hello", hello_path)
      concat " "
      concat link_to("Bye", goodbye_path)
    end